ecs-agent/api/ecs/client/ecs_client_test.go

```diff
@@ -798,6 +800,44 @@ func TestDiscoverNilServiceConnectEndpoint(t *testing.T) {
 		"Expected error getting service connect endpoint with old response")
 }
 
+func TestDiscoverSystemLogsEndpoint(t *testing.T) {
+	ctrl := gomock.NewController(t)
```

Comment:
nit: These test can be refactored into table-driven tests.

Reply:
Valid point, this whole file can be improved, but I'll defer test refactoring to another PR.